frontend/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+function Button(props) {
+  const { propname } = props;
+  return (
+    <button
+      type="submit"
+      className="items-center px-5 py-2.5 text-sm font-medium text-center text-red-700 rounded-lg border-2 border-red-500 focus:ring-1 focus:ring-red-600 hover:bg-rose-200 "
+    >
+      {propname}
+    </button>
+  );
+}
+
+export default Button;

```

Comment:
Add EOF

Reply:
Is there only one type of button ? If there is multiple types of buttons (ex. primary, error/danger, success) please make it dynamic.